recipes/plotly_express/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "plotly_express" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ff73a41ce02fb43d1d8e8fa131ef3e6589857349ca216b941b8f3f862bce0278
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - numpy >=1.11
```

Comment:
Are all these dependencies really required for building? I guess they are only needed in `run`.

Reply:
It's hit or miss whether people write setup.py to work properly without all of the dependencies (and without specifying `--no-deps`), but let's see.